README.md

```diff
@@ -32,7 +32,6 @@ The Web API lets you access the legislation [Parameters](./parameters.md) and [V
 
 - To explore the OpenFisca-France Web API services, use the [French Legistation Explorer](https://legislation.openfisca.fr/)
 - To query the Openfisca Web API in your app, see the [Web API endpoints description](./openfisca-web-api/endpoints.md)
-- To make sure the OpenFisca-France Web API (and all the related services) are still up and running, check out our [status page](https://status.openfisca.fr/)
```

Comment:
> This status page still seems to exist, but is frankly outdated. Ok with removing the reference while we update it, but eventually, shouldn't the documentation mention the status page somewhere?

Reply:
Oh oh oh you're right! In my cleaning frenzy I mixed up the “build status” page which is totally irrelevant if it even exists at all anymore, and the online services status page. Thanks for pointing that out!

I say we still go with removal because the description is outdated anyway, but I'm all for adding it again, giving proper visibility to that status page, and making it international!

Reply:
https://github.com/openfisca/openfisca.org/issues/33